File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1011_basico.js
import { question } from "readline-sync";

function main() {
    const r = get_number();

    const pi = 3.14159;
    let volume = Calcular_Volume(pi, r);

    print(`VOLUME = ${volume.toFixed(3)}`);
}

function get_number() {
    return Number(question(`Raio: `));
}

function print(texto) {
    return console.log(texto);
}

function Calcular_Volume(pi, r) {
    return ((4/3) * pi * (r ** 3));
}

main();

File: Exercicios_ED_I/Bee/1011_basico.cpp
#include <stdio.h>
#include <iostream>
using namespace std;

main() {
    double raio = 0.0;
    double pi = 3.14159;

    cin >> raio;
    
    double volume = ((4.0 / 3.0) * pi * (raio * raio * raio));

    printf("VOLUME = %.3lf\n", volume);
}